client/my-sites/domains/domain-management/settings/index.tsx

```diff
@@ -48,50 +59,88 @@ const Settings = ( props: SettingsPageProps ): JSX.Element => {
+						isLoadingPurchase={ isLoadingPurchase }
+					/>
+				</Accordion>
+			);
+		}
+	};
+
+	const renderMainContent = () => {
+		// TODO: If it's a registered domain or transfer and the domain's registrar is in maintenance, show maintenance card
+		return <>{ renderDetailsSection() }</>;
+	};
+
 	const renderSettingsCards = () => (
 		<>
-			<DomainEmailInfoCard selectedSite={ props.selectedSite } domain={ props.domain } />
-			<DomainTransferInfoCard selectedSite={ props.selectedSite } domain={ props.domain } />
-			<DomainDeleteInfoCard selectedSite={ props.selectedSite } domain={ props.domain } />
+			<DomainEmailInfoCard selectedSite={ selectedSite } domain={ domain } />
+			<DomainTransferInfoCard selectedSite={ selectedSite } domain={ domain } />
+			<DomainDeleteInfoCard selectedSite={ selectedSite } domain={ domain } />
 		</>
 	);
 
+	if ( ! domain ) {
+		// TODO: Update this placeholder
+		return <DomainMainPlaceholder breadcrumbs={ renderBreadcrumbs } />;
+	}
+
 	return (
-		<Main wideLayout>
+		<Main wideLayout className="settings">
+			{ selectedSite.ID && ! purchase && <QuerySitePurchases siteId={ selectedSite.ID } /> }
 			<BodySectionCssClass bodyClass={ [ 'edit__body-white' ] } />
 			{ renderBreadcrumbs() }
-			<SettingsHeader domain={ props.domain } />
-			<TwoColumnsLayout
-				content={
-					<>
-						Page goes here.
-						{ /* Placeholder to test accordion */ }
-						<div style={ { marginTop: '30px' } }>
-							<Accordion
-								title="First element title"
-								subtitle="First element subtitle"
-								expanded={ true }
-							>
-								<div>Component placeholder: this one is exapanded by default</div>
-							</Accordion>
-							<Accordion title="Second element title" subtitle="Second element subtitle">
-								<div>Component placeholder: this one i'snt exapanded by default</div>
-							</Accordion>
-						</div>
-					</>
-				}
-				sidebar={ renderSettingsCards() }
-			/>
+			<SettingsHeader domain={ domain } />
+			<TwoColumnsLayout content={ renderMainContent() } sidebar={ renderSettingsCards() } />
 		</Main>
 	);
 };
 
 export default connect(
-	( state, ownProps: SettingsPagePassedProps ): SettingsPageConnectedProps => {
+	( state, ownProps: SettingsPageProps ): SettingsPageConnectedProps => {
+		const domain = ownProps.domains && getSelectedDomain( ownProps );
+		const subscriptionId = domain && domain.subscriptionId;
+		const currentUserId = getCurrentUserId( state );
+		const purchase = subscriptionId
+			? getByPurchaseId( state, parseInt( subscriptionId, 10 ) )
+			: null;
+
 		return {
-			domain: getSelectedDomain( ownProps )!,
 			currentRoute: getCurrentRoute( state ),
-			hasDomainOnlySite: Boolean( isDomainOnlySite( state, ownProps.selectedSite!.ID ) ),
+			domain: getSelectedDomain( ownProps )!,
+			isLoadingPurchase:
+				isFetchingSitePurchases( state ) || ! hasLoadedSitePurchasesFromServer( state ),
+			purchase: purchase && purchase.userId === currentUserId ? purchase : null,
```

Comment:
Optionally you could update the Purchase type adding `userId` prop (client/lib/purchases/types.ts) to avoid TS linter error.

Reply:
Good call, updated the `Purchase` type